packages/manager/modules/pci/src/projects/project/kubernetes/details/node-pool/nodes/add/translations/Messages_en_GB.json

```diff
@@ -1,5 +1,5 @@
 {
-  "kube_nodes_add": "Add node",
```

Comment:
Not in https://github.com/ovh/manager/blob/develop/packages/manager/modules/pci/src/projects/project/kubernetes/details/node-pool/nodes/add/translations/Messages_fr_FR.json